src/MLJ.jl

```diff
@@ -47,40 +34,40 @@ function MMI.update(
     while cache[:info][:nrounds] < model.nrounds
       grow_evotree!(fitresult, cache, model)
     end
-    report = (features=A.names,)
+    report = (features=Tables.schema(A).names,)
   else
     fitresult, cache, report = fit(model, verbosity, A, y, w)
   end
   return fitresult, cache, report
 end
 
 function predict(::EvoTreeRegressor, fitresult, A)
-  pred = vec(predict(fitresult, A.matrix))
+  pred = predict(fitresult, A)
```

Comment:
This `pred` is a *vector*, and not a `1 x n` matrix, right? Otherwise, we need `vec` and I remember this being an issue in past.

Reply:
Similar queries apply at lines 50, 55, 60, 65, 70 below.

Reply:
Yes EvoTrees' `predict` has been updated in the meantime to return vector rather than matrix for Regressor: https://github.com/Evovest/EvoTrees.jl/blob/535b0c47e7acb726e5d6591588deeceecadbacbe/src/predict.jl#L106